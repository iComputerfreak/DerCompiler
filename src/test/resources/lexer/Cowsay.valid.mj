 _________________________________________
< Caution: Keep out of reach of children. >
 -----------------------------------------
                           ^__^   /
                   _______/(oo)  /
                 /(       /(__)
                * | w----||
                  ||     ||
