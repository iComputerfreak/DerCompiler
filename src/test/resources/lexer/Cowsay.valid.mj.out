identifier _________________________________________
<
identifier Caution
:
identifier Keep
identifier out
identifier of
identifier reach
identifier of
identifier children
.
>
--
--
--
--
--
--
--
--
--
--
--
--
--
--
--
--
--
--
--
--
-
^
identifier __
^
/
identifier _______
/
(
identifier oo
)
/
/
(
/
(
identifier __
)
*
|
identifier w
--
--
||
||
||
EOF
